db-connector/src/main/java/uk/nhs/adaptors/connector/dao/ImmunizationSnomedCTDao.java

```diff
@@ -9,7 +9,7 @@
 public interface ImmunizationSnomedCTDao {
     @SqlQuery("select_immunization_concept_id")
     @UseClasspathSqlLocator
-    ImmunizationSnomedCT getImmunizationSnomednUsingConceptId(@Bind("conceptId") String conceptId);
+    ImmunizationSnomedCT getImmunizationSnomedUsingConceptOrDescriptionId(@Bind("sctid") String sctId);
```

Comment:
Hmm, `sctid` feels a bit lazy. As a developer joining the project and taking a look at that it would mean nothing to them. Let's rename this to `snomedId`?